tests/GUIUtils.lua

```diff
@@ -99,18 +99,21 @@ describe("Tests the GUI utilities as far as possible without mudlet", function()
         { "\27[38;2;120;134;94m", "<120,134,94:>"},
         { "\27[48;2;85;250;33m", "<:85,250,33>"},
         { "\27[38;2;120;134;94;48;2;85;250;33m", "<120,134,94:85,250,33>"},
```

Comment:
Ah, we do not **_currently_** support it, see `void TBuffer::translateToPlainText( std::string & s )`:

                            if( mIsHighColorMode )
                            {
                                switch( tag )
                                {
                                case 5: // Indexed 256 color mode
                                    mWaitingForHighColorCode = true;
                                    break;
                                case 2: // 24Bit RGB color mode
    // TODO:
    //                                mWaitingFor24BitColor = true;
    //                                break;
                                case 4: // 24Bit CYMB color mode
                                case 3: // 24Bit CYM color mode
                                case 1: // "Transparent" mode
                                case 0: // "Application defined" mode
                                    qWarning() << "TBuffer::translateToPlainText(...) Warning unhandled ANSI SGR 38/48 type color code encountered, first parameter is:" << tag;
                                    break;
                                default:
                                    qWarning() << "TBuffer::translateToPlainText(...) Warning unknown ANSI SGR 38/48 type color code encountered, first parameter is:" << tag;
                                    break;
                                }


                                continue;
                            }